locales/en-US/messages.properties

```diff
@@ -1 +1,9 @@
+unable_to_contact_login_server=Unable to contact login server.<br/> Refresh the page to try again.
+loading_message=Loading&hellip;
+your_project=Your Projects
+log_out=Log Out
+sign_in=Sign in
+or=or
+sign_up=Sign Up
+hi_username=Hi, {name}
```

Comment:
I think either `hi_name` or `hi_username` would make the most sense. The pattern we're doing is string+var